content/docs/command-reference/cache/dir.md

```diff
@@ -24,22 +24,31 @@ Unlike doing so with `dvc config cache`, `dvc cache dir` transform paths
 **relative to the config file location**. However, if the `value` provided is an
 absolute path, then it's preserved as it is.
 
-If no path `value` is provided to this command, it prints the path for current
-cache directory.
+When reading (`value` is not provided) the value is read from the system,
+global, repo and local configuration files by default, and options `--system`,
+`--global`, `--repo`, and `--local` can be used to tell the command to read from
+only that location.
+
+When writing (`value` is provided or `--unset` is used), the new value is
+written to the projects configuration file by default (`.dvc/config`), and
+options `--system`, `--global` and `--local` can be used to tell the command to
+write to that location (you can say `--repo` but that is the default).
 
 ## Options
 
-- `--global` - modify a global config file (e.g. `~/.config/dvc/config`) instead
-  of the project's `.dvc/config`.
+- `--global` - use a global config file (e.g. `~/.config/dvc/config`) instead of
+  the project's `.dvc/config`.
```

Comment:
For example here we're still calling it "project". We should probably be consistent to avoid confusions. Repo and project don't always refer to the same thing (depends on the context).